packages/backend/src/managers/apiServer.ts

```diff
@@ -144,6 +151,31 @@ export class ApiServer implements Disposable {
     app.use('/api', router);
     app.use('/spec', this.getSpec.bind(this));
 
+    const options = {};
```

Comment:
why declaring options here? if we don't do put anything in it, the option is optional 

```ts
serveFiles(swaggerDoc?: JsonObject, opts?: SwaggerUiOptions): RequestHandler[];
```

let's remove it